packages/glsp-playwright-core/src/integration/wait.fixes.ts

```diff
@@ -0,0 +1,9 @@
+// TODO: Workaround because of VSCode - VSCode does not really work well with page.waitForFunction
+export async function waitForFunction(cb: () => Promise<boolean>): Promise<void> {
+    let finished = await cb();
+
+    while (!finished) {
+        await new Promise(res => setTimeout(() => res(''), 100));
```

Comment:
Would be cool if we could give the timeout as a parameter to the function with 100 being the default. I don't think we need an overall timeout until we give up because I assume that playwright has an option like that, right?

Reply:
yes, in the `playwright.config.ts` you can provide a testcase restricted max `time` everything from start to end can take.